scripts/generate_thumbnail.py

```diff
@@ -32,12 +32,16 @@
 with open(data_path, 'rt', encoding='utf-8') as f:
```

Comment:
Quick question: If you are copying most of the functions from this file to `src/thumbnail/main_generate_thumbnail.py`, then wouldn't we have code redundancy?

Reply:
Like I said, I forgot to remove my changes in this file.
I didn't find an easy way to import the script inside `TSHScoreboardWidget` so I moved your script inside `src/thumbnail/main_generate_thumbnail.py`, made some changes (create a methode `generate()`, change some file path etc)

So there is currently redundancy so you can generate the thumbnail directly via `scripts/genereate_thumbnail.bat` and via the UI
It may be possible to keep only one script but I don't really know how, maybe change `scripts/genereate_thumbnail.bat` to call the script inside `src/thumbnail/main_generate_thumbnail.py` or the other way (but I didn't find how)

Reply:
Fair enough, my opinion would be that we can keep the redundancy for now and once we have all of the features implemented in the UI we can remove the manual script altogether

There is a way to [import modules from the parent directory](https://www.geeksforgeeks.org/python-import-from-parent-directory/) so the script `scripts/genereate_thumbnail.py` could import the contents of `src/thumbnail/main_generate_thumbnail.py`, but I'm not sure it's worth the effort